modules/module_group_not_unix.go

```diff
@@ -1,18 +1,21 @@
-//go:build windows
+//go:build darwin || windows
 
 package modules
 
 import (
 	"fmt"
 	"log"
+	"runtime"
 
 	"github.com/skx/marionette/environment"
 )
 
 // Execute is part of the module-api, and is invoked to run a rule.
 func (g *GroupModule) Execute(env *environment.Environment, args map[string]interface{}) (bool, error) {
 
-	log.Printf("[ERROR] the 'group' module is not implemented upon Windows")
+	message := "the 'group' module is not implemented up this platform"
```

Comment:
I think that should be either:

* .. on this platform
* .. upon this platform

But `up` is definitely wrong :)

Reply:
Oops, fixed.